projects/nlp/junit_test5.py

```diff
@@ -45,7 +45,7 @@ def runExperiment(args):
                     [d for d in dataSet if d[2]%100==0],
                     categorySize=5,
                     verbosity=args.verbosity)
-  printRankResults("JUnit4", ranks)
+  printRankResults("JUnit5", ranks)
```

Comment:
There are a few bugs in junit_test5.py. I think the above block of code should be:
```
  allRanks, avgRanks, avgStats  = testModel(model,
                    [d for d in dataSet if d[2]%100==0],
                    categorySize=6,
                    verbosity=args.verbosity)
  printRankResults("JUnit5", avgRanks, avgStats)
```